data/test/macros/wml_unit_test_macros.cfg

```diff
@@ -75,3 +75,90 @@
         [/then]
     [/if]
 #enddef
+
+#define ATTACK_AND_VALIDATE STARTX STARTY X1 Y1 X2 Y2 DAMAGE1
```

Comment:
This should have documentation, but it also seems to be reinventing wheels.

* `COMMON_KEEP_A_B_UNIT_TEST` would avoid the movement part
* `[test_do_attack_by_id]` would use unit ids instead of needing hardcoded coordinates

Reply:
Updated.